benchmarks/benchmark_runner.py

```diff
@@ -155,9 +219,10 @@ def main() -> None:
       model_name=benchmark_model,
       software_config=v6e_env_configs,
       hardware_config=v6e_256_configs,
+      num_steps=options.num_steps,
   )
 
-  xpk_benchmark_runner(cluster_config, [model_runner])
+  xpk_benchmark_runner(cluster_config, [model_runner], options.xpk_path)
```

Comment:
will need to add default value for xpk path for `xpk_benchmark_runner `

Reply:
Done